contracts/src/v0.8/ccip/libraries/Internal.sol

```diff
@@ -75,21 +75,25 @@ library Internal {
     uint256 proofFlagBits;
   }
 
-  /// @dev Any2EVMRampMessage struct has 10 fields, including 3 variable unnested arrays (data, receiver and tokenAmounts).
+  /// @dev Any2EVMRampMessage struct has 10 fields, including 3 variable unnested arrays (sender, data and tokenAmounts).
   /// Each variable array takes 1 more slot to store its length.
   /// When abi encoded, excluding array contents,
   /// Any2EVMMessage takes up a fixed number of 13 slots, 32 bytes each.
+  /// Assume 1 slot for sender
   /// For structs that contain arrays, 1 more slot is added to the front, reaching a total of 14.
   /// The fixed bytes does not cover struct data (this is represented by MESSAGE_FIXED_BYTES_PER_TOKEN)
-  uint256 public constant MESSAGE_FIXED_BYTES = 32 * 14;
+  uint256 public constant MESSAGE_FIXED_BYTES = 32 * 15;
 
-  /// @dev Each token transfer adds 1 RampTokenAmount
```

Comment:
nit: I think this comment is useful to mention which struct we are referring to (needs to be updated to `Any2EVMTokenTransfer`)

Reply:
This is a deleted line, updating in the follow up comments